Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -150,17 +150,21 @@ theorem mk_zpow (a : G) (n : ℤ) : ((a ^ n : G) : Q) = (a : Q) ^ n :=
 
 @[to_additive (attr := simp)] lemma map_mk'_self : N.map (mk' N) = ⊥ := by aesop
 
+omit nN in
+private theorem lift_aux (φ : G →* M) (HN : N ≤ φ.ker) {x y : G} (h : x⁻¹ * y ∈ N) :
```

Comment:
I think there's a non-private theorem hiding here, as
```lean
theorem con_le_ker_of_le_ker (φ : G →* M) (HN : N ≤ φ.ker) :
    QuotientGroup.con N ≤ Con.ker φ := by
  intro x y h
```

Reply:
Or perhaps even:
```lean

theorem con_le_conKer_iff (φ : G →* M) :
    QuotientGroup.con N ≤ Con.ker φ ↔ N ≤ φ.ker := by
  constructor
  · intro hN x hx
    specialize @hN x 1
    simp_all [QuotientGroup.con, leftRel_apply]
  · intro HN x y h
    -- adapt existing proof
  ```

Reply:
Perhaps this is nicer to prove by first showing that `con` is an order isomorphism, with the reverse direction being the set `{x | r x 1}`

Reply:
Ok, so I did that in a new PR #23269. The application of the new result to the proof of `QuotientGroup.lift` is not as nice as one would like since `QuotientGroup.con f.ker` and `Con.ker f` are not equal but `QuotientGroup.con f.ker x y = Con.ker f y x`

Reply:
Oh, I realized what I did wrong. I'll fix that later.

Reply:
The new version of #23269 should be okay. Sorry for the noise.